### ISSUE TITLE
Resend offered after verification

### ISSUE THREAD

**Issue by Kebap** [Mudlet/mudlet-package-repo#72]
After Login I see "Successfully logged in!"
and menu contains "Resend Email Verification"
but I already verified it before.

**Comment by vadi2** [comment]
Yeah, it shows up once and then goes away on refresh. I believe that's because that part of the page is generated _before_ the verification happens. Needs to be fixed.

**Comment by Kebap** [comment]
It also lists "Logout" alongside, so that part is true.
